src/pages/patientView/patientHeader/PatientHeader.tsx

```diff
@@ -40,15 +34,15 @@ export default class PatientHeader extends React.Component<IPatientHeaderProps,
         );
     }
 
-    private getPopoverPatient(patient: typeof _ClinicalInformationData.patient) {
+    private getPopoverPatient(patient: ClinicalInformationData['patient']) {
         return patient && (
```

Comment:
You mean runtime or ts? Is there a legitimate reason for patient to be undefined?  If not, it should crash or throw an error.  If so, then it should be handled in the control flow. Sorry if I'm misunderstanding.